tremor-cli/src/test/assert.rs

```diff
@@ -345,6 +348,8 @@ pub(crate) fn process_filebased_asserts(
                 contains: None,
                 doesnt_contain: None,
                 equals_file: None,
+                is_empty: None,
```

Comment:
is_empty and is_not_empty seem to be the same just inverted. It looks like they can be reduced to a single flag, halfing the needed code and preventing possible confusion.